function testcase ( ) { try { } catch ( e ) { return false } } runTestCase ( testcase ) ;